util/hookable/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+#
+# (c) Copyright IBM Corp. 2016
+#
+#  This program and the accompanying materials are made available
+#  under the terms of the Eclipse Public License v1.0 and
+#  Apache License v2.0 which accompanies this distribution.
+#
+#      The Eclipse Public License is available at
+#      http://www.eclipse.org/legal/epl-v10.html
+#
+#      The Apache License v2.0 is available at
+#      http://www.opensource.org/licenses/apache2.0.php
+#
+# Contributors:
+#    Multiple authors (IBM Corp.) - initial implementation and documentation
+###############################################################################
+
+#TODO there is a bunch of stuff with vpaths, presumably for some extensibility reasons
```

Comment:
this TODO comment doesn't really describe an action, could you please make it more clear what needs to be done here?